runtime/onert/core/src/ir/Shape.cc

```diff
@@ -75,8 +75,8 @@ uint64_t Shape::num_elements() const
 {
   // if dimension is 0, it means unspecified and cannot calculate the total number of elements
```

Comment:
@hyunsik-yoon This comment is incorrect. I will update this and more related `Shape` in another PR since it already has 3 approvals.